### PR TITLE
RC review: add jupyter as it is needed for doc

### PR DESCRIPTION
**Pull request by rcaneill**

### CODE REVIEW

**Comment by jbousquin** [comment]
Agree jupyter notebooks are needed for demos. Moving from notebooks to jupyterlabs as suggested will take some further investigation but I'm not opposed.

**Comment by jbousquin** [comment]
I had a chance to dig into this - jupyterlab doesn't have much additional dependencies but a couple of conda env builds seemed to have issues where the same env with notebook was fine. If there is a compelling reason for labs over notebooks I can try to resolve those but otherwise tempted to stick with what works.

**Comment by rcaneill** [comment]
I believe that you could stick to notebook instead of jupyterlab, but I don’t see it in the requirements-dev.txt

File: requirements_dev.txt [hunk 1]
```diff
@@ -4,4 +4,5 @@ sphinx
 sphinx_rtd_theme
 sphinxcontrib-spelling
 nbsphinx
-nbsphinx_link
\ No newline at end of file
+nbsphinx_link
+jupyterlab
```
Comment (rcaneill) [inline requirements_dev.txt:8]:
```suggestion
notebook
```

**Review by jbousquin — APPROVED** [review]

**Pull request merged**